From a410e8396f8badbf8b15689701bf3e7fef84ab04 Mon Sep 17 00:00:00 2001
From: Julian Brown <julian.brown@cpanel.net>
Date: Thu, 24 Sep 2020 15:49:34 -0500
Subject: [PATCH] shutdown libxml2 warning

---
 lib/nokogiri/version.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/nokogiri/version.rb b/lib/nokogiri/version.rb
index f919cc5..c738ab2 100644
--- a/lib/nokogiri/version.rb
+++ b/lib/nokogiri/version.rb
@@ -35,6 +35,7 @@ module Nokogiri
     end
 
     def warnings
+      return []
       return [] unless libxml2?
 
       if compiled_parser_version != loaded_parser_version
-- 
2.25.1

